owncloudApp/src/main/java/com/owncloud/android/presentation/files/filelist/FileListAdapter.kt

```diff
@@ -203,10 +214,27 @@ class FileListAdapter(
                         it.Filename.text = file.fileName
                         it.fileListSize.text = DisplayUtils.bytesToHumanReadable(file.length, context)
                         it.fileListLastMod.text = DisplayUtils.getRelativeTimestamp(context, file.modificationTimestamp)
-                        it.fileListPath.apply {
-                            text = file.remotePath
-                            isVisible = !fileListOption.isAllFiles()
+                        if (fileListOption.isAvailableOffline()) {
```

Comment:
Ok, I'll change it, but instead of `!fileListOption.isAllFiles()`, I will add `fileListOption.isSharedByLink()` because there's no need to show the path in the Shares section for oCIS users (files will be always located in the root folder of the Shares space).